src/app/common/filter-text/filter-text.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+
+@Injectable()
+export class FilterTextService {
+  constructor() {}
+
+  filter(data: string, props: Array<string>, originalList: Array<any>) {
+    let filteredList = [];
+
+    if (data && props && originalList) {
+      data = data.toLowerCase();
+
+      originalList.filter(item => {
+        let match = false;
+
+        for (const prop of props) {
+
+          console.log('prop: ' + JSON.stringify(prop));
+
+          if (item[prop].toString().toLowerCase().indexOf(data) > -1) {
+            console.log('match found: ' + JSON.stringify(item[prop]));
+            match = true;
+
+            filteredList.push(item);
+
+            console.log('match: ' + JSON.stringify(match));
+            console.log('item: ' + JSON.stringify(item));
+            break;
+          }
+
+          console.log('filteredList: ' + JSON.stringify(filteredList));
```

Comment:
Not a problem, I completely agree. I'll remove them. Btw, we used to have that logger from hawtio that @gashcrumb added in, not sure where it went. Would be nice to have env-specific logging for when forgetful devs like me forget to remove logs.